### PR TITLE
Doc: Update Russian locales

### PR DESCRIPTION
**Pull request by bropines**
Update russian locales

### CODE REVIEW

**Review by GaiZhenbiao — APPROVED** [review]

**Review by Keldos-Li — APPROVED** [review]

File: readme/README_ru.md [hunk 1]
```diff
@@ -73,24 +73,24 @@
...
+### 🤖 Системный промт
 - Установка предпосылок через системное сообщение позволяет эффективно играть роль персонажа;
 - Чуаньху Чат предоставляет набор системных шаблонов, нажмите "Загрузить шаблон системного сообщения", затем выберите необходимый шаблон ниже.
 
 ### 💬 Обычный диалог
 - Если ответ не удовлетворяет вас, можно попробовать снова с помощью кнопки "Перегенерировать" или просто удалить этот раунд диалога;
-- Поле ввода поддерживает перенос строки, нажмите `Shift` + `Enter`, чтобы сделать перенос строки;
-- В поле ввода можно использовать клавиши `↑` и `↓`, чтобы быстро переключаться в истории отправки;
+- Поле ввода поддерживает перенос строки, нажмите <kbd>Shift</kbd> + <kbd>Enter</kbd>, чтобы сделать перенос строки;
+- В поле ввода можно использовать клавиши <kbd>↑</kbd> и <kbd>↓</kbd>, чтобы быстро переключаться в истории отправки;
```
Comment (Keldos-Li) [inline readme/README_ru.md:88]: Wait, maybe we should unify whether to use the `< kbd > `tag or just ````.

Comment (Keldos-Li) [reply readme/README_ru.md:88]: Oh, 说晚了 😰

Comment (bropines) [reply readme/README_ru.md:88]: > Oh, 说晚了 😰

KBD looks more clear to me. I think it's better to use it when describing "buttons"

Comment (MZhao-ouo) [reply readme/README_ru.md:88]: Yes, KBD is easier to read. However, some markdown editors fail to recognize KBD tags correctly, resulting in formatting errors. To ensure the stability, we have decided not to use them.🥲

Comment (bropines) [reply readme/README_ru.md:88]: > Yes, KBD is easier to read. However, some markdown editors fail to recognize KBD tags correctly, resulting in formatting errors. To ensure the stability, we have decided not to use them.🥲

I understand, well, if that’s the case, you can remove them in the Russian version. I just use the built-in markdown viewer in VSCode and edit it as text.

Comment (Keldos-Li) [reply readme/README_ru.md:88]: > Yes, KBD is easier to read. However, some markdown editors fail to recognize KBD tags correctly, resulting in formatting errors. To ensure the stability, we have decided not to use them.🥲

No, I would vote for KBD. The most important function of readme is to show when others browse github projects, so readability comes first.

**Pull request merged**